### PR TITLE
fix: restore IRTF non-meeting groups to agenda

### LINKED ISSUE #5010: Proceedings ignores "notmeet" irtf sessions
### Describe the issue

PR #5005 inadvertently changed the handling of irtf sessions set as 'notmeet'. These are no longer shown on the proceedings. They are left off whether or not they have materials attached to them. Previously, the 'notmeet' flag was ignored for irtf sessions.

For IETF-115, this results in anrw and qirg not being listed on the proceedings.

Need to decide how to display these sessions (and whether they should be included in the first place).

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by rjsparks**
Fixes #5010

### CODE REVIEW

File: ietf/templates/meeting/proceedings.html [hunk 1]
```diff
@@ -173,11 +173,39 @@ <h2 class="mt-5" id="irtf">
...
+                <tr>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for entry in irtf.not_meeting %}{% if entry.sessions_with_materials %}
```
Comment (jennifer-richards) [inline ietf/templates/meeting/proceedings.html:203]: This needs to be updated to match the corresponding change in #5065. (That is, it should check `entry.has_materials` instead)

**Review by jennifer-richards — CHANGES_REQUESTED** [review]
Need to merge in main to pick up #5065 and update the template to match the change there.

File: ietf/templates/meeting/proceedings.html (hunk 1 above)
Comment (rjsparks) [reply ietf/templates/meeting/proceedings.html:203]: hoop - old copy of main :(

**Review by jennifer-richards — APPROVED** [review]

File: ietf/templates/meeting/proceedings.html (hunk 1 above)
Comment (ChiefGhostZero0x1) [reply ietf/templates/meeting/proceedings.html]: {"id_str":"429799239393476608","retweet_count":0,"display_text_range":[0,109],"is_quote_status":false,"source":"<a href=\"http:\/\/www.apple.com\" rel=\"nofollow\">iOS<\/a>","lang":"en","place":null,"full_text":"Check out my classic car in #CSRClassics for iPhone! It's FREE! http:\/\/t.co\/ttSFt3gn6K http:\/\/t.co\/ZYm0vHNdZR","favorite_count":0,"user":{"profile_sidebar_border_color":"C0DEED","favourites_count":54,"geo_enabled":true,"profile_text_color":"333333","listed_count":2,"profile_background_image_url_https":"https:\/\/abs.twimg.com\/images\/themes\/theme1\/bg.png","is_translation_enabled":false,"verified":false,"follow_request_sent":false,"notifications":false,"profile_background_image_url":"http:\/\/abs.twimg.com\/images\/themes\/theme1\/bg.png","screen_name":"AkaChiefy","withheld_in_countries":[],"profile_banner_url":"https:\/\/pbs.twimg.com\/profile_banners\/2319153741\/1640789674","utc_offset":null,"lang":null,"entities":{"url":{"urls":[{"display_url":"github.com\/website","indices":[0,23],"url":"https:\/\/t.co\/wPTpJaJxqb","expanded_url":"http:\/\/github.com\/website"}]},"description":{"urls":[]}},"url":"https:\/\/t.co\/wPTpJaJxqb","friends_count":427,"followers_count":49,"profile_sidebar_fill_color":"DDEEF6","translator_type":"none","profile_use_background_image":true,"statuses_count":1295,"created_at":"Sun Feb 02 02:03:19 +0000 2014","profile_background_color":"C0DEED","default_profile_image":false,"contributors_enabled":false,"has_extended_profile":true,"location":"Phoenix, AZ","time_zone":null,"profile_image_url":"http:\/\/pbs.twimg.com\/profile_images\/1476205004205690881\/n3SRc7qU_normal.jpg","id_str":"2319153741","is_translator":false,"profile_link_color":"1DA1F2","profile_background_tile":false,"id":2319153741,"protected":false,"default_profile":true,"description":"Fun loving dog dad of two amazing Australian Cattle Dogs - Chief the Blue Heeler and Koda the Red Heeler. perpetual energy","profile_image_url_https":"https:\/\/pbs.twimg.com\/profile_images\/1476205004205690881\/n3SRc7qU_normal.jpg","following":false,"name":"Todd Gifford"},"in_reply_to_user_id":null,"coordinates":null,"in_reply_to_status_id":null,"geo":null,"id":4.2979923939347661e+17,"in_reply_to_user_id_str":null,"extended_entities":{"media":[{"expanded_url":"https:\/\/twitter.com\/tgifford37\/status\/429799239393476608\/photo\/1","indices":[87,109],"id":4.2979923921314611e+17,"url":"http:\/\/t.co\/ZYm0vHNdZR","id_str":"429799239213146113","media_url_https":"https:\/\/pbs.twimg.com\/media\/Bfb0IYIIYAEFPBy.jpg","media_url":"http:\/\/pbs.twimg.com\/media\/Bfb0IYIIYAEFPBy.jpg","display_url":"pic.twitter.com\/ZYm0vHNdZR","type":"photo","sizes":{"thumb":{"h":150,"resize":"crop","w":150},"medium":{"h":640,"resize":"fit","w":1136},"small":{"h":383,"resize":"fit","w":680},"large":{"h":640,"resize":"fit","w":1136}}}]},"entities":{"media":[{"expanded_url":"https:\/\/twitter.com\/tgifford37\/status\/429799239393476608\/photo\/1","indices":[87,109],"id":4.2979923921314611e+17,"url":"http:\/\/t.co\/ZYm0vHNdZR","id_str":"429799239213146113","media_url_https":"https:\/\/pbs.twimg.com\/media\/Bfb0IYIIYAEFPBy.jpg","media_url":"http:\/\/pbs.twimg.com\/media\/Bfb0IYIIYAEFPBy.jpg","display_url":"pic.twitter.com\/ZYm0vHNdZR","type":"photo","sizes":{"medium":{"h":640,"resize":"fit","w":1136},"large":{"h":640,"resize":"fit","w":1136},"thumb":{"h":150,"resize":"crop","w":150},"small":{"h":383,"resize":"fit","w":680}}}],"hashtags":[{"indices":[28,40],"text":"CSRClassics"}],"urls":[{"indices":[64,86],"url":"http:\/\/t.co\/ttSFt3gn6K","expanded_url":"http:\/\/nmgam.es\/cct","display_url":"nmgam.es\/cct"}],"symbols":[],"user_mentions":[]},"truncated":false,"contributors":null,"in_reply_to_status_id_str":null,"favorited":false,"retweeted":false,"in_reply_to_screen_name":null,"possibly_sensitive":false,"created_at":"Sun Feb 02 02:11:38 +0000 2014"}

**Pull request merged**